### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by Hardikchhatbar2006**

### CODE REVIEW

File: Readme.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello, I'm doing this web project for my learning purposes...
```
Comment (AdiechaHK) [inline Readme.txt]: please change this.

Comment (AdiechaHK) [reply Readme.txt]: this file should be `.md` file not `.txt` file.

File: assets/scss/Readme.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+The .scss (Sass) files are only available in the pro version.
```
Comment (AdiechaHK) [inline assets/scss/Readme.txt]: this is also not needed.

File: forms/contact.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    'port' => '587'
+  );  
+  */
+
+  $contact->add_message( $_POST['name'], 'From');
+  $contact->add_message( $_POST['email'], 'Email');
+  $contact->add_message( $_POST['message'], 'Message', 10);
+
+  echo $contact->send();
+?>
```
Comment (AdiechaHK) [inline forms/contact.php]: This file is not needed at all.

**Review by AdiechaHK — CHANGES_REQUESTED** [review]
check my review

**Comment by Hardikchhatbar2006** [comment]
done

File: assets/js/main.js [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+        slidesPerView: 1,
+        spaceBetween: 20
+      },
+
+      1200: {
+        slidesPerView: 3,
+        spaceBetween: 20
+      }
+    }
+  });
```
Comment (AdiechaHK) [inline assets/js/main.js]: remove this if not needed

File: assets/js/main.js [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+  //   window.location.href = "mailto:hardikchahtbar2006@gmail.com?subject="+SubjectLine+"&body="+body;
+  //   document.getElementById('EmailButton').onclick(sendMail());
+  // } 
+
+
+
+  /**
+   * Initiate Pure Counter 
+   */
+  new PureCounter();
```
Comment (AdiechaHK) [inline assets/js/main.js:259]: remove if not needed

**Review by AdiechaHK — CHANGES_REQUESTED** [review]

File: assets/js/main.js (hunk 4 above)
Comment (Hardikchhatbar2006) [reply assets/js/main.js]: This is used in portfolio section too so it is mandatory

File: assets/js/main.js (hunk 5 above)
Comment (Hardikchhatbar2006) [reply assets/js/main.js:259]: this is also needed for the counter that we have made by our logic

**Review by AdiechaHK — APPROVED** [review]

**Pull request merged**